test/src/webExtension.spec.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2025 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import path from 'node:path';
+
+import expect from 'expect';
+
+import {getTestState, launch, setupTestBrowserHooks} from './mocha-utils.js';
+
+const EXTENSION_PATH = path.join(__dirname, '/../assets/simple-extension');
+const EXPECTED_ID = 'mbljndkcfjhaffohbnmoedabegpolpmd';
+
+describe('webExtension', function () {
+  setupTestBrowserHooks();
+
+  it.only('can install and uninstall an extension', async () => {
+    const {defaultBrowserOptions, isChrome} = await getTestState({
+      skipLaunch: true,
+    });
+
+    const options = Object.assign({}, defaultBrowserOptions);
+
+    if (isChrome) {
+      options.ignoreDefaultArgs = ['--disable-extensions'];
```

Comment:
let's add an option to launch called `enableExtensions` that way we can manage args for the user

Reply:
As discussed elsewhere, let's do this in a follow-up PR.